### PR TITLE
Add simplified Chinese version for the README doc

### PR DESCRIPTION
**Pull request by lynklody**
- Added a simplified Chinese README file @ the home dir
- Added a language selection bar in the main README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +55,7 @@ not straightforward.
 ## Very Much Frequently Asked Questions ❓
 
 ### How much doge can exist? – So many puppies! 🐕
-Early 2015 (approximately a year and a half after release) there were
+Early 2015 (approximately a year and a half after release) there will be
```
Comment (jimmyt1988) [inline README.md:58]: This reads correct before this edit considering it's now past tense. It's worth reverting this hunk.

**Review by jimmyt1988 — COMMENTED** [review]
Cool! Nice one! I guess we'll need a few people who can speak Chinese to approve this one! Much amaze, Many wow!

**Comment by rnicoll** [comment]
Yeah I did spot checks with Google translate but ideally we'd have someone review before we merge 🤔

File: README_zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+**测试链（testnet）和私有网络（regtest）模式**
+
+如果你在通过因特网测试跨机代码，在测试链上运行`-testnet`来运行“开始使用狗狗币”。
+
+如果你在测试单机代码，用`-regtest`运行。
+在回归测试模式下，区块可以随时被按需制造出来；参见qa/rpc-tests/。
+
+**DEBUG_LOCKORDER**
+
+狗狗币核心是一个多线程应用，死锁和其他多线程问题都会很难找到原因。通过`-DDEBUG_LOCKORDER`编译 (`configure CXXFLAGS="-DDEBUG_LOCKORDER -g"`)会在运行时插入检查语句来检测哪些进程被锁定并且把警告加入debug.log文件中，如果在代码中发现了不一致的地方。
```
Comment (jimthunderbird) [inline README_zh_CN.md:125]: Seems like this should be changed to:

狗狗币核心是一个多线程应用，碰到死锁和其他多线程问题都会很难找到原因。如果在代码中发现了不一致的地方，使用`-DDEBUG_LOCKORDER`编译 (`configure CXXFLAGS="-DDEBUG_LOCKORDER -g"`)会在运行时插入检查语句来检测哪些进程被锁定，并把警告加入debug.log文件中。

Comment (lynklody) [reply README_zh_CN.md:125]: This sounds better. Thanks for the review!

File: README.md (hunk 1 above)
Comment (lynklody) [reply README.md]: Just realize my previous reply to your comment was placed in a wrong place. Basically I was saying, thank you for pointing it out. I copied the original README file from the master branch a few days back and the English text was "will be". I should have develop with dev-1.14.4 directly. Thanks!

**Review by ZilinL — COMMENTED** [review]
Good job!
This change may make it easier for Chinese readers to understand.

狗狗币采用了一种简化版的scrypt秘钥派生函数作为它的工作量证明，目标时间为每分钟产出一个区块，每产出一个区块后都会进行难度值调整。区块奖励是固定的，且每产出100,000个区块就会减半。但从第600,000个区块开始，每产出一个区块的奖励会固定在10,000个狗狗币。

起初，我们设想了一种不同的支付方式，由Mersenne Twister伪随机数生成器在0到区块计划相应的最大区块奖励中生成一个随机数作为区块奖励。

然而从第145,000个区块开始，为了防止大矿池钻空子只挖高奖励的区块，这种方案就不再采用了。与此同时，难度值调整也从每4小时变为每个区块产生（每分钟）更改一次，使用的是DigiByte Coin开发团队的一个算法，以减少狗狗币网络中哈希率突然增大或减少带来的影响。

**Comment by lynklody** [comment]
> Good job!
> This change may make it easier for Chinese readers to understand.
> 

Thanks for the revision. I modified the translated README file accordingly. I'm a newbie to cryptocurrency and I learned something through your translation!

**Review by ZilinL — APPROVED** [review]

**Comment by lynklody** [comment]
Thanks for the approval. Can we merge now?

**Comment by angelcervantesa** [comment]
How can I request a doc for approval for a different language?

**Review by patricklodder — APPROVED** [review]
utACK

**Comment by lynklody** [comment]
Thanks Patrick! @patricklodder

**Pull request merged**